### ISSUE TITLE
Issues with torbox rclone mount

### ISSUE THREAD

**Issue by Bayharbourbutcher1** [sureshfizzy/CineSync#35]
If you set the env variable for rclone mount to true in cinesync config, there are multiple errors thrown in the torbox rclone logs. ![Screenshot_20241211_191503_Hermit.jpg](https://github.com/user-attachments/assets/08aaa3ca-4f3e-467c-995b-edb5ef0bcea7)

Not sure if it can cause an issue but just sharing.

**Comment by sureshfizzy** [comment]
Is it a usenet upload ? I remember seeing 405 issue on usenet provider.

**Comment by Bayharbourbutcher1** [comment]
No, usenet is not being used at all. This happens when cinesync tries to place 'version.txt' file in the rclone mount.

**Comment by sureshfizzy** [comment]
is there any guide for mounting torbox, I have'nt used it so far

**Comment by sureshfizzy** [comment]
> This is a sample docker compose for rclone:
> 
> Services: torbox: image: rclone/rclone:latest container_name: torbox restart: unless-stopped environment: TZ: Europe/Berlin PUID: 1000 PGID: 1001 UMASK: 002 volumes: - /mnt/remote/torbox:/data:rshared - ./rclone.conf:/config/rclone/rclone.conf - /mnt:/mnt cap_add: - SYS_ADMIN security_opt: - apparmor:unconfined devices: - /dev/fuse:/dev/fuse:rwm command: "mount torbox: /data --cache-dir /mnt/remote/cache/torbox --vfs-fast-fingerprint --rc --rc-addr :5573 --rc-no-auth --allow-other --allow-non-empty --async-read=true --dir-cache-time=15s --buffer-size=48M --poll-interval=15s --use-mmap --vfs-read-ahead=96M --vfs-read-chunk-size=32M --vfs-read-chunk-size-limit=32M --vfs-cache-max-age=672h --vfs-cache-mode=full --vfs-cache-poll-interval=15s --vfs-cache-max-size=200G --timeout=10m --umask=002 --log-level INFO --vfs-refresh --gid=1001 --uid=1000"
> 
> This is the rclone conf for torbox:
> 
> [torbox] type = webdav url = https://webdav.torbox.app vendor = other user = torbox-email pass = rclone-encrypted-pass
> 
> I followed a guide in the riven discord.

Thanks, I was able to reproduce it. But initally i could not find any version.txt  in torbox when it is mounted. CineSync is trying to create one but since torbox is not allowing any uploads through mount, it is failing.

**Comment by Bayharbourbutcher1** [comment]
Yes, thats it and the logs keep filling with the upload errors.

**Comment by sureshfizzy** [comment]
> Yes, thats it and the logs keep filling with the upload errors.

This issue appears to be quite tricky when dealing with WebDAV mounts, particularly during creation or checking of a `version.txt` file. On WebDAV mounts, even if the mount gets disconnected, the local environment can still detect the disconnection. However, inside Docker, the mount remains marked as connected, and everything continues to function as expected without triggering any errors. This behavior prevents the mount flag from properly reflecting the "unavailable" status when the mount is no longer accessible.

**Comment by sureshfizzy** [comment]
@Bayharbourbutcher1, this issue has been addressed and confirmed to be working with other WebDAV mounts. However, it won't work with Torbox due to the limitations of their mount system, which is quite poor.
